src/09-workorders/work-orders.apib

```diff
@@ -23,6 +23,12 @@ Add a draft work order.
                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
                + quantity: 3 (number, required)
        + location (Address, optional)
+       + work_hours (array, optional)
```

Comment:
Should we rename this to `time` so that we have time & materials? Would be nice 😬

Reply:
🤯